kong/tools/database_cache.lua

```diff
@@ -100,6 +101,11 @@ function _M.ssl_data(api_id)
   return CACHE_KEYS.SSL.."/"..api_id
 end
 
+function _M.hmacauth_credential_key(username)
```

Comment:
Make this a local function in the file it is use since it is not an entity from the core.